CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] - 2022-04-07
```

Comment:
It would be interesting to mention the breaking change, and how to migrate to version 2.
Thanks @paulnta!

Reply:
I added one small breaking change (removal of `forbidden` parameter on `validate-copyleft` job). Otherwise, the rest works exactly like before.

Reply:
Yes, and now with the addition of the explanation in the changelog it's 👌🏼